base/Dockerfile

```diff
@@ -4,19 +4,16 @@ ARG JITSI_RELEASE=stable
 
 ENV S6_BEHAVIOUR_IF_STAGE2_FAILS=2
 
-ADD https://github.com/just-containers/s6-overlay/releases/download/v1.22.1.0/s6-overlay-amd64.tar.gz /tmp/s6-overlay.tar.gz
-ADD https://download.jitsi.org/jitsi-key.gpg.key /tmp/jitsi.key
-ADD https://github.com/subchen/frep/releases/download/v1.3.5/frep-1.3.5-linux-amd64 /usr/bin/frep
-
 COPY rootfs /
 
 RUN \
-	tar xfz /tmp/s6-overlay.tar.gz -C / && \
-	rm -f /tmp/*.tar.gz && \
 	apt-dpkg-wrap apt-get update && \
 	apt-dpkg-wrap apt-get install -y apt-transport-https apt-utils ca-certificates gnupg && \
-	apt-key add /tmp/jitsi.key && \
-	rm -f /tmp/jitsi.key && \
+	DEBIAN_FRONTEND=noninteractive apt-get install -y wget && \
```

Comment:
Can you please use apt-dpkg-wrap instead? https://github.com/jitsi/docker-jitsi-meet/blob/master/base/rootfs/usr/bin/apt-dpkg-wrap

Reply:
Have done. 
To be consistent, 'apt-get remove' has also been modified.